rundeck/resource_job_test.go

```diff
@@ -163,6 +194,54 @@ resource "rundeck_job" "test" {
 }
 `
 
+const testAccJobConfig_cmd_nodefilter = `
+resource "rundeck_project" "test" {
+  name = "terraform-acc-test-job"
+  description = "parent project for job acceptance tests"
+
+  resource_model_source {
+    type = "file"
+    config = {
+        format = "resourcexml"
+        file = "/tmp/terraform-acc-tests.xml"
+    }
+  }
+}
+resource "rundeck_job" "test" {
+  project_name = "${rundeck_project.test.name}"
+  name = "basic-job-with-node-filter"
+  description = "A basic job"
+  execution_enabled = true
+  node_filter_query = "example"
+  allow_concurrent_executions = true
+  max_thread_count = 1
+  rank_order = "ascending"
+	schedule = "0 0 12 * * * *"
+	schedule_enabled = true
+  option {
+    name = "foo"
+    default_value = "bar"
+  }
+  command {
+    job {
+      name = "Other Job Name"
+      run_for_each_node = true
+      nodefilters = {
+        filter: "name: tacobell"
```

Comment:
I'm not sure why this works, the colon fails to parse in an actual TF run haha.

Reply:
I think it works in tf 0.12. Maybe it doesn’t work in 0.11?